### PR TITLE
remove assets cache as a linked directory

### PR DESCRIPTION
**Pull request by tampakis**
Compile assets on each deploy

### CODE REVIEW

**Comment by jpstroop** [comment]
@terrellt, could you review?

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -34,4 +34,6 @@
 //= require modernizr
 
 //= require orangelight
+//= require availability
```
Comment (terrellt) [inline app/assets/javascripts/application.js]: Is the explicit requires just a matter of preference? The require tree statement is enough for the asset pipeline.

Comment (tampakis) [inline app/assets/javascripts/application.js]: It didn't work without these explicit requires. Maybe there's a setting that in the config that we don't have enabled?

Comment (terrellt) [inline app/assets/javascripts/application.js]: That's strange...if it works in dev it should compile in prod.

**Pull request merged**